javascript/modules/client/test/integration/client.test.ts

```diff
@@ -131,32 +134,31 @@ describe("Client", () => {
   });
 
   describe("DAO Creation", () => {
-    test.todo("Should estimate gas fees for creating a DAO");
-    // it("Should estimate gas fees for creating a DAO", async () => {
-    //   const context = new Context(contextParamsLocalChain);
-    //   const client = new Client(context);
-
-    //   const daoCreationParams: ICreateParams = {
-    //     daoConfig: {
-    //       name: "ERC20VotingDAO_" + Math.floor(Math.random() * 9999) + 1,
-    //       metadata: "0x1234",
-    //     },
-    //     plugins: [
-    //       { id: "0x1234", data: "0x1234" },
-    //     ],
-    //     gsnForwarder: Wallet.createRandom().address,
-    //   };
-
-    //   const gasFeesEstimation = await client.estimation.create(
-    //     daoCreationParams,
-    //   );
-
-    //   expect(typeof gasFeesEstimation).toEqual("object");
-    //   expect(typeof gasFeesEstimation.average).toEqual("bigint");
-    //   expect(typeof gasFeesEstimation.max).toEqual("bigint");
-    //   expect(typeof gasFeesEstimation.max).toBeGreaterThan(BigInt(0));
-    //   expect(gasFeesEstimation.max).toBeGreaterThan(gasFeesEstimation.average);
-    // });
+    it("Should estimate gas fees for creating a DAO", async () => {
+      const context = new Context(contextParamsLocalChain);
+      const client = new Client(context);
+
+      const daoCreationParams: ICreateParams = {
+        daoConfig: {
+          name: "ERC20VotingDAO_" + Math.floor(Math.random() * 9999) + 1,
+          metadata: "0x1234",
+        },
+        plugins: [
+          { id: "0x1234", data: "0x1234" },
+        ],
+        gsnForwarder: Wallet.createRandom().address,
```

Comment:
Could we remove this parameter from the definition? We're not going to have meta transactions in this version/release, at all.

Reply:
Are you referring to the `plugins` or to the `gsnForwarder`

Reply:
the GSN forwarder :+1: